doc/features/tuning-policies/README.md

```diff
@@ -157,16 +157,175 @@ unexpected error, invoked in the following situations:
     - When the contacted host replies with an error, such as `overloaded`, `isBootstrapping`, `serverError`, etc. In 
     this case, the error is instance of `ResponseError`
 
-The [operation info][OperationInfo], passed as a parameter to the retry policy methods, exposes the `query` and query 
-`options` as properties.
+A default and base retry policy is included, along with `IdempotenceAwareRetryPolicy` that considers query idempotence. The default implementation doesn't use `IdempotenceAwareRetryPolicy`, in order to use it you should wrap the default policy with `IdempotenceAwareRetryPolicy`:
+
+``` js
+ client = new cassandra.Client({
```

Comment:
It would be nice to avoid using the `cassandra` qualifier / module name whenever possible, that way it can be safer for the user to copy and paste (ie: users can import `Client` directly).

Reply:
OK, thanks. 
I still want to show where those objects are in the package so I included a `require` statements. I can say that when I looked how to do those things it took me some time to find all the policies.